dfastmi/batch/core.py

```diff
@@ -483,21 +483,22 @@ def _get_root_dir(rootdir: Path) -> Path:
     return rootdir
 
 
-def count_discharges(discharges: Vector) -> int:
+def count_discharges(apply_q: Tuple[bool, bool, bool]) -> int:
```

Comment:
Please use BoolVector

Reply:
Done. However, `apply_q` is a `Tuple` and `BoolVector` is a shorthand for a `List` type ... I see that this is mixed up at other locations as well.
I've made the use of `BoolVector` now consistent, but created a new issue DFAST-236 to address the type inconsistency. I needed tuples in the past for stricter type checking by the `pypy` static analyzer, but lists should also work.